setup.py

```diff
@@ -12,7 +12,7 @@
     'pytest-cov>=2.8.1,<3',
     'coveralls[yaml]>=1.8.2,<2',
     'pytest-xdist>=1.30.0,<2',
-    'eth-tester[py-evm]==0.1.0b39',
+    'eth-tester[py-evm]>=0.2.0b1,<0.3',
```

Comment:
FWIW, the latest released version is `v0.2.0b3`

Reply:
Haha, well I'll be waiting for `0.2.0b4` to get the fix!

Reply:
How bad is it for you folks if the release doesn't come out till Monday?

Reply:
Not bad at all, tests still work under <0.2

Reply:
Okay, since Istanbul is now the default VM, I bumped the minor version number. Give 'v0.3.0-beta.1` a try.